### ISSUE TITLE
Git branch name in the status bar

### ISSUE THREAD

**Issue by stefano-cherchi** [jupyterlab/jupyterlab-git#1100]
## Is your feature request related to a problem? Please describe.
While you're working on the notebook you can't see which branch you're working on, unless you select the Git extension tab. I noticed that this can be confusing and lead to errors especially for developers with limited experience on git.

## Describe the solution you'd like
It would be nice to have the name of the current branch in the bottom status bar, next to the git status icon, like in VSCode or other IDEs.

- Python package version:
```
$ pip show jupyterlab-git
Name: jupyterlab-git
Version: 0.34.0
Summary: A JupyterLab extension for version control using git
Home-page: https://github.com/jupyterlab/jupyterlab-git
Author: Jupyter Development Team
Author-email: None
License: BSD-3-Clause
Location: /app/anaconda3-2021.05/lib/python3.8/site-packages
Requires: pexpect, nbformat, jupyter-server, nbdime, packaging
Required-by:
```

- Extension version:
```
$ jupyter labextension list
/app/anaconda3-2021.05/lib/python3.8/site-packages/jupyter_server/transutils.py:13: FutureWarning: The alias `_()` will be deprecated. Use `_i18n()` instead.
  warnings.warn(warn_msg, FutureWarning)
JupyterLab v3.0.14
/app/anaconda3-2021.05/share/jupyter/labextensions
        jupyterlab-sparkmonitor v3.0.1 enabled OK (python, jupyterlab_sparkmonitor)
        nbdime-jupyterlab v2.1.0 enabled OK
        jupyterlab-execute-time v2.0.5 enabled OK (python, jupyterlab_execute_time)
        jupyterlab-plotly v5.6.0 enabled OK
        @jupyter-widgets/jupyterlab-manager v3.0.0 enabled OK (python, jupyterlab_widgets)
        @jupyterlab/git v0.34.0 enabled OK (python, jupyterlab-git)

Other labextensions (built into JupyterLab)
   app dir: /app/anaconda3-2021.05/share/jupyter/lab
```

- Git version:
```
$ git --version
git version 2.32.0
```

- Operating System and its version:
```
$ cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.3 (Maipo)
```

**Comment by fcollonval** [comment]
Thanks for the suggestion. I can provide some pointers to implement this. I would only suggest to make it configurable as some users like not too crowded status bar.
